README.md

```diff
@@ -1 +1,3 @@
-# OOP-training
\ No newline at end of file
+# OOP SCHOOL LIBRARY
+
+This is a school library
```

Comment:
- [ ] It will be great to add a descriptive readme file. This will ensure that your project Is professional. To do this, kindly add instructions on how to run your project and other details that are important for any user to know